contracts/test/plugin/extensions/governance/addresslist.ts

```diff
@@ -110,145 +106,133 @@ describe('AddresslistMock', function () {
+        .withArgs(carol.address);
 
       // Verify that the address list has not changed
       await ethers.provider.send('evm_mine', []);
-      expect(await addresslist.isListed(signers[0].address)).to.equal(true);
-      expect(await addresslist.isListed(signers[2].address)).to.equal(true);
+      expect(await addresslist.isListed(alice.address)).to.equal(true);
+      expect(await addresslist.isListed(carol.address)).to.equal(true);
       expect(await addresslist.addresslistLength()).to.equal(2);
     });
 
     it('reverts if the array of new addresses to be added contains an address multiple times', async () => {
-      await expect(
-        addresslist.addAddresses([signers[0].address, signers[0].address])
-      )
+      await expect(addresslist.addAddresses([alice.address, alice.address]))
         .to.be.revertedWithCustomError(addresslist, 'InvalidAddresslistUpdate')
-        .withArgs(signers[0].address);
+        .withArgs(alice.address);
     });
   });
 
   context('removeAddresses', function () {
     it('removes existing addresses from the address list', async () => {
-      await addresslist.addAddresses([signers[0].address, signers[1].address]);
+      await addresslist.addAddresses([alice.address, bob.address]);
       await ethers.provider.send('evm_mine', []);
 
-      expect(await addresslist.isListed(signers[0].address)).to.equal(true);
-      expect(await addresslist.isListed(signers[1].address)).to.equal(true);
+      expect(await addresslist.isListed(alice.address)).to.equal(true);
+      expect(await addresslist.isListed(bob.address)).to.equal(true);
       expect(await addresslist.addresslistLength()).to.equal(2);
 
-      await expect(
-        addresslist.removeAddresses([signers[0].address, signers[1].address])
-      ).not.to.be.reverted;
+      await expect(addresslist.removeAddresses([alice.address, bob.address]))
+        .not.to.be.reverted;
 
       await ethers.provider.send('evm_mine', []);
 
-      expect(await addresslist.isListed(signers[0].address)).to.equal(false);
-      expect(await addresslist.isListed(signers[1].address)).to.equal(false);
+      expect(await addresslist.isListed(alice.address)).to.equal(false);
+      expect(await addresslist.isListed(bob.address)).to.equal(false);
       expect(await addresslist.addresslistLength()).to.equal(0);
     });
 
     it('reverts removal if an address is not listed', async () => {
-      await addresslist.addAddresses([signers[0].address, signers[1].address]);
+      await addresslist.addAddresses([alice.address, bob.address]);
       await ethers.provider.send('evm_mine', []);
       expect(await addresslist.addresslistLength()).to.equal(2);
 
       // try to remove signer[1] and signer[2], the latter of which is currently not listed
-      await expect(
-        addresslist.removeAddresses([signers[1].address, signers[2].address])
-      )
+      await expect(addresslist.removeAddresses([bob.address, carol.address]))
```

Comment:
should we use bob & carol naming in the comment as well? if not, why not?

Reply:
Fixed in 59c2c6d9780d9a28049c772e57da2e2ffcaebdbe